Codes/Clean Data/getOrderSize.R

```diff
@@ -0,0 +1,17 @@
+function(price, sz, indexSub, string) {
+  
+  
+  orszSVol <- sz[indexSub] #order size in terms of share volume
+  orszDVol <-
+    sz[indexSub] * price[indexSub] #order size in terms of dollar volume

```

Comment:
What is the price? Is it market price or midquote or the limit price?

Reply:
In this context, "price" is the price at which the order is submitted.